### PR TITLE
wishlistReducer and wishlistContext Added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### All Wishlist functionality added

- Product added in wishlist page after clicking on add to wishlist.
- If product already present in wishlist then Go to wishlist button showing else Add to wishlist button showing.
- User can add product in cart from wishlist if its not present in cart.
- User can remove product from wishlist.
- Wishlist products count updating according action.

https://user-images.githubusercontent.com/59019169/161486543-b9b9243c-29c5-4708-a188-395d1945bcbf.mp4

ction.

### CODE REVIEW

File: src/CSS/button.css [hunk 1]
```diff
@@ -16,7 +16,7 @@
 .btn-outline-primary{
     border: 2px solid var(--primary-dark-color) !important;
     color: var(--primary-dark-color) !important;
-    background-color: var(--black) !important;
+    background-color: var(--white);
     font-size: 1.5rem !important;
 }
```
Comment (prathmeshjagtap) [inline src/CSS/button.css:21]: avoid using important

**Review by prathmeshjagtap — APPROVED** [review]
Code is Well structured and Clean just avoid  using Important in css

**Review by harshkotkar5557 — APPROVED** [review]
Looks good to me

File: src/CSS/button.css (hunk 1 above)
Comment (Mahi24-prog) [reply src/CSS/button.css:21]: ok

**Pull request merged**